django/utils/http.py

```diff
@@ -3,8 +3,9 @@
 import unicodedata
 from binascii import Error as BinasciiError
 from datetime import datetime, timezone
-from email.utils import formatdate
-from urllib.parse import quote, unquote
+from email.message import Message
+from email.utils import collapse_rfc2231_value, formatdate
+from urllib.parse import quote
```

Comment:
I would move this import along with the existing `from urllib.parse import urlsplit`, in general we rather not have multiple lines importing from the same module.

Reply:
It was actually a fix of pre-commit, and pipeline fails when I try to get imports in one line. Three imports are listed in alphabetical order.

I do not think I can actually remove `urlencode as original_urlencode` construction, so it may be the best version of imports.

Reply:
Yes, if you drop the `as` import then isort (under pre-commit) will re-combine the imports.

Reply:
Dropping the `as` would be possible with some refactor but that would be a separated change that the one in this branch. For now I think we will have to live with the multiple imports for the same lib. Thanks both!